tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -296,9 +296,8 @@ def initial_mirror_config(self, mirror2, poolname, imagename, **kw):
             p.spawn(mirror2.create_pool, poolname=poolname)
 
         self.create_image(imagespec=imagespec, size=imagesize)
-        if kw.get("image_feature"):
-            image_feature = kw.get("image_feature")
-            self.image_feature_enable(imagespec=imagespec, image_feature=image_feature)
```

Comment:
I think we need not remove these lines of code, we will just check if mode is pool or mirrormode is journal and if kw["image_feature"] = journaling is present, then we will enable that feature using these removed lines of code. If image_feature not passed and still mode is pool or mirrormode is journal, then we enable journaling feature by default.
@rkhambad Hope I did not confuse you.

Reply:
@manasagowri, added the required changes as per discussion. Right now not included this test module with both ec-pool and rep-pool configuration, as we have different task created for the same.

Reply:
let me know if this is the expected.

Reply:
@manasagowri  done the requested changes as discuss in peer review meeting.